schema/me/conversation.js

```diff
@@ -0,0 +1,38 @@
+import impulse from "../../lib/loaders/impulse"
+import gravity from "../../lib/loaders/gravity"
+
+import { GraphQLString, GraphQLNonNull } from "graphql"
+
+import { ConversationType } from "./conversations"
+
+const { IMPULSE_APPLICATION_ID } = process.env
+
+export default {
+  type: ConversationType,
+  description: "A conversation",
+  args: {
+    id: {
+      type: new GraphQLNonNull(GraphQLString),
+      description: "The ID of the Conversation",
+    },
+  },
+  resolve: (root, options, request, { rootValue: { accessToken, userID } }) => {
+    if (!accessToken) return null
+
+    const id = options.id
```

Comment:
probably can replace `options` in the method signature above with just `{ id }` to avoid this boilerplate line.